compute/gpu/how-to/create-manage-gpu-instance.mdx

```diff
@@ -30,7 +30,7 @@ Scaleway GPU Instances are virtual compute instances equipped with dedicated hig
     - Choose an **Availability Zone**, which is the geographical region where your GPU Instance will be deployed. GPU Instances are currently available in PAR-1 and/or PAR-2, depending on the GPU Instance type.
     - Select **GPU** as the **Instance type**, and then choose the type of GPU Instance you want. Different types have different prices, processing power, memory, storage options and bandwidth. 
     - Choose an **Image** to run on your GPU Instance. The following images are available, depending on your GPU Instance type:
-      - **Ubuntu Focal GPU OS 11**: our latest GPU OS image, with preinstalled Nvidia drivers and an Nvidia Docker environment. Other software is not pre-installed, as you are expected to use Docker to launch a working environment suitable for your needs. You can build your own container, use an official Docker image, or choose from our range of [Scaleway Docker images](/compute/gpu/reference-content/docker-images/), each of which has CUDA installed and is customized for different purposes.
+      - **Ubuntu Jammy GPU OS 12**: our latest GPU OS image, with preinstalled Nvidia drivers and an Nvidia Docker environment. Other software is not pre-installed, as you are expected to use Docker to launch a working environment suitable for your needs. You can build your own container, use an official Docker image, or choose from our range of [Scaleway Docker images](/compute/gpu/reference-content/docker-images/), each of which has CUDA installed and is customized for different purposes.
     - Add **Volumes**. This is an optional step. Volumes are storage spaces used by your Instances. You can leave the default settings of a minimum local storage, or choose to add more [Block](/compute/instances/concepts/#block-volumes) and/or reduce [Local](/compute/instances/concepts/#local-volumes) Storage to your GPU Instance. You can also choose which volume to run the OS from.
     - Enter a **Name** for your GPU Instance, or leave the randomly generated name in place. Optionally, you can also add [tags](/compute/instances/concepts/#tags) to help you organize your GPU Instance. 
     - Click **Advanced Options** if you want to configure a [flexible IP](/compute/instances/concepts/#flexible-ip), a local bootscript or a [cloud-init configuration](/compute/instances/concepts/#cloud-init). Otherwise, leave these options at their default values. 
```

Comment:
```suggestion
    - Click **Advanced options** if you want to configure a [flexible IP](/compute/instances/concepts/#flexible-ip), a local bootscript or a [cloud-init configuration](/compute/instances/concepts/#cloud-init). Otherwise, leave these options at their default values. 
```